config/env_schema.hjson

```diff
@@ -70,9 +70,18 @@
         "MIVIDEO": {
             "type": "object",
             "properties": {
-                "service_account_json_filename": {"type": "string"}
+                "default_last_timestamp": {"type": "string"}, # should be "date-time", but fails!
+                "udp_service_account_json_filename": {"type": "string"},
+                "kaltura_partner_id": {"type": "integer"},
+                "kaltura_user_id": {"type": "string"}, # should be "email", but fails!
```

Comment:
@ssciolla, you're more familiar with the HJSON validation than I.  Maybe you know something about this…

* I specified `"date-time"` and `"email"` for the field types where appropriate, but the validation still fails, even when I know the value types in the input are correct.
* Validation order seems to be random.  Without changing the inputs or the code, when I run the program multiple times, sometimes it will fail on the `date-time` validation and sometimes it would fail on the `email` one.

Reply:
1) My understanding is that JSON doesn't have those data types (https://www.json.org/json-en.html, https://json-schema.org/understanding-json-schema/reference/type.html). Do you have a different understanding?

2) I'm not sure why you're getting different ordering there. I'll do a little investigation. This doesn't seem like a big issue to me though; you fix one error and then you fix the next.

Reply:
Oh, sorry, I'm now seeing what you're referring to here: https://json-schema.org/understanding-json-schema/reference/string.html#format

I'll take a look.

Reply:
Following up to my direct message. JSON Schema supports a `format` keyword in schemas, but it seems not all implementations support it. The `jsonschema` Python package does seem to support it optionally; see the docs here: https://python-jsonschema.readthedocs.io/en/latest/validate/#validating-formats

It looks like you may need to modify the `validate` call in `environ.py` and install another package to validate `date-time`. Once everything is in place, I think you should be able to do something like this:

```
{
   "type": "string",
   "format": "date-time"
}
```

Reply:
I've resolved this with commit fa3ec24.